Assets/Scripts/Turn-System/CombatManager.cs

```diff
@@ -9,6 +9,9 @@ public class CombatManager : MonoBehaviour
     private int _currentTurn;
     private CombatantType _activeType;
 
+    //Store the body part data list
+    public List<BodyPartData> _bodyPartDataList;
```

Comment:
This should be part of the Protagonist, shouldn't really be in the combat manager.